### PR TITLE
Feature/wb 7 world builder starting page panel

### PR DESCRIPTION
**Pull request by Daboo1998**
Created the starting page of World builder.

Issue: https://momentum.nifty.pm/l/_X!iYrV87n

### CODE REVIEW

File: src/ui-kit/molecules/Steps/Steps.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <styled.StepContainerWrapper key={step}>
+          <styled.StepContainer>
+            <styled.Step className={cn(currentStep && index < currentStep && 'completed')}>
+              {currentStep === index && <styled.SelectedFill />}
+              {!!currentStep && index < currentStep && index + 1}
+            </styled.Step>
+            <styled.Label label={step} type="h3" />
+          </styled.StepContainer>
+          {index < steps.length - 1 && (
+            <styled.LikingLine className={cn(currentStep && index < currentStep && 'completed')} />
```
[the remark this answers is not in the document]
Comment (Daboo1998) [reply src/ui-kit/molecules/Steps/Steps.tsx:24]: Hahaha yep, seems I really like the line 😂

File: src/scenes/worldBuilder/WorldBuilder.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+const WorldBuilder: FC = () => {
+  return (
+    <styled.Container>
+      <styled.Background src={background} alt="background" />
+      <styled.BackgroundFilter />
+
+      <styled.PageContainer>
+        <Switch>
+          {createRoutesByConfig(WORLD_BUILDER_ROUTES)}
+          <Redirect from={ROUTES.worldBuilder.base} to={ROUTES.worldBuilder.start} />
```
Comment (PAKovalev) [inline src/scenes/worldBuilder/WorldBuilder.tsx:21]: Remove from

File: src/scenes/worldBuilder/WorldBuilder.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import {ROUTES} from 'core/constants';
+import background from 'static/images/worldBuilder.png';
+
+import {WORLD_BUILDER_ROUTES} from './WorldBuilder.routes';
+import * as styled from './WorldBuilder.styled';
+
+const WorldBuilder: FC = () => {
+  return (
+    <styled.Container>
+      <styled.Background src={background} alt="background" />
```
Comment (PAKovalev) [inline src/scenes/worldBuilder/WorldBuilder.tsx:15]: remove alt

File: src/shared/services/i18n/locales/en-gb.ts [hunk 4]
```diff
@@ -443,7 +448,9 @@ export const enGb = {
       close: 'Close',
       inviteToStage: 'Invite to stage',
       ready: 'Ready',
-      inviteThisPersonToTheStage: 'Invite this person to the stage?'
+      inviteThisPersonToTheStage: 'Invite this person to the stage?',
+      logIn: 'Log In',
```
Comment (PAKovalev) [inline src/shared/services/i18n/locales/en-gb.ts:452]: move to actions section

File: src/static/styles/variables.css [hunk 5]
```diff
@@ -65,7 +65,8 @@
   --default-progress-bar: rgba(240, 145, 2, 1);
 
   /** Typography **/
-  --font-size-xxl: 25px;
+  --font-size-xxxl: 25px;
```
Comment (PAKovalev) [inline src/static/styles/variables.css:68]: Update StoryBook

File: src/ui-kit/molecules/Steps/Steps.tsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+import cn from 'classnames';
+
+import * as styled from './Steps.styled';
+
+interface PropsInterface {
+  currentStep?: number;
+  steps: string[];
+}
+
+const Steps: FC<PropsInterface> = ({currentStep, steps}) => {
```
Comment (PAKovalev) [inline src/ui-kit/molecules/Steps/Steps.tsx:11]: Add to story book + add testid attr

File: src/scenes/worldBuilder/WorldBuilder.styled.ts [hunk 7]
```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+  width: 100vw;
+  height: 100vh;
+`;
+
+export const PageContainer = styled.div`
```
Comment (PAKovalev) [inline src/scenes/worldBuilder/WorldBuilder.styled.ts:8]: make a ui-kit component with `PageContainer` and `BackgroundFilter`. We can reuse this.

**Review by PAKovalev — CHANGES_REQUESTED** [review]

File: src/shared/services/i18n/locales/en-gb.ts (hunk 4 above)
Comment (Daboo1998) [reply src/shared/services/i18n/locales/en-gb.ts:452]: It is already in it 😉

File: src/scenes/storyBook/pages/MoleculesPage/MoleculesPage.tsx [hunk 8]
```diff
@@ -394,6 +396,31 @@ const MoleculesPage: FC = () => {
           <ToolbarIcon title="StarOn" icon="starOn" onClick={() => {}} />
           <ToolbarIcon title="Chat" icon="chat" onClick={() => {}} />
         </styled.Section>
+        <styled.Name>
+          <Heading label="Component «Steps»" type="h2" align="left" />
+        </styled.Name>
+        <styled.Section>
+          <TabBar
+            tabs={[
+              {id: 'undefined', title: 'undefined', label: 'Step undefined'},
```
Comment (PAKovalev) [inline src/scenes/storyBook/pages/MoleculesPage/MoleculesPage.tsx:405]: Can we remove this component?

File: src/scenes/worldBuilder/pages/StartPage/StartPage.tsx [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+  const {t} = useTranslation();
+  const history = useHistory();
+
+  return (
+    <styled.Container>
+      <styled.Spacer />
+      <styled.Information>
+        <styled.LogoContainer>
+          <styled.Logo src={logo} />
+          <styled.Title label="World Builder" transform="uppercase" type="h1" />
```
Comment (PAKovalev) [inline src/scenes/worldBuilder/pages/StartPage/StartPage.tsx:25]: i18n

**Review by PAKovalev — CHANGES_REQUESTED** [review]

File: src/scenes/storyBook/pages/MoleculesPage/MoleculesPage.tsx (hunk 8 above)
Comment (Daboo1998) [reply src/scenes/storyBook/pages/MoleculesPage/MoleculesPage.tsx:405]: Well, then you cannot really test how the steps behave in different conditions.

**Review by aminbksi — APPROVED** [review]

**Pull request merged**